Server/Utilities/TextInputFirstResponderProvider.m

```diff
@@ -36,7 +36,7 @@ - (XCUIElement *)firstResponder {
     XCUIElement *firstResponder = nil;
     for (NSNumber *number in self.elementTypes) {
         XCUIElementType type = (XCUIElementType)[number unsignedIntegerValue];
-        XCUIElementQuery *firstResponderQuery = [application descendantsMatchingType:type];
+        XCUIElementQuery *firstResponderQuery = [(id)application descendantsMatchingType:type];
```

Comment:
pls check if we can avoid this change

Reply:
Fixed. Anyway we should cast XCUIApplication to XCUIElement.